pandas/tests/test_frame.py

```diff
@@ -5545,6 +5545,8 @@ def test_itertuples(self):
         dfaa = df[['a', 'a']]
         self.assertEqual(list(dfaa.itertuples()), [(0, 1, 1), (1, 2, 2), (2, 3, 3)])
 
+        self.assertEqual(type(next(df.itertuples(name=None))), tuple)
```

Comment:
I disagree: testing is exactly the context where `type(spam) is spam` is what you do if you want to check for exact type and not for ancestry. I would use `self.assertIs` but AFAIR this was added in Python 2.7.

What do you propose to use as check here?